egs/commonvoice/asr1/run.sh

```diff
@@ -100,11 +100,9 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
     for x in ${train_set} ${recog_set}; do
       # Remove features with too long frames in training data
       max_len=3000
```

Comment:
Are `max_len=3000` and `max_chars=10000` appropriate?
Especially `max_chars=10000` looks very strange.

Reply:
I mean to set a large threshold on character-level to follow the previous results (with only a 3000 frame threshold).

Reply:
why not using default one?
Some corpus includes very long sentences (by mistakes), and 10,000 will break training.

Reply:
The default one should be enough, I will change it by next commit